resotolib/resotolib/utils.py

```diff
@@ -514,3 +514,19 @@ def error(self, message: str) -> NoReturn:
     def exit(self, status: int = 0, message: Optional[str] = None) -> NoReturn:
         msg = message if message else "unknown"
         raise AttributeError(f"Could not parse arguments: {msg}")
+
+
+def replace_env_vars(elem: JsonElement) -> JsonElement:
```

Comment:
I would opt to pass the env vars as dict explicitly. A) easier to test B) we have control over the env vars that get passed. Please add a test.